.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,74 @@
+name: native-metrics CI
+
+on:
+  push:
+    branches:
+    - open-source-main
+  pull_request:
+    branches:
+    - '**'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install Dependencies
+      run: npm ci
+    - name: Rebuild
```

Comment:
i'm assuming the linting is javascript only? i wonder if we can avoid the build step for linting.

otherwise, even though we only need to lint one version... it might be faster to include in teh other jobs cause linting will be much faster than building.

Reply:
Oh nice!  Good thinking.